src/utils/ardrive.test.ts

```diff
@@ -171,9 +158,8 @@ describe('ArDrive class', () => {
 			});
 
 			const actual = await arDrive.estimateAndAssertCostOfFolderUpload(stubPublicFileTransactionData);
-			expect(actual).to.deep.equal({
-				metaDataBaseReward: '147'
-			});
+			// TODO: Bummer to lose deep equal verification
```

Comment:
How so? I didn't get this comment

Reply:
Stale comment. Will remove.

Reply:
Actually it's not stale. Equality on the safe types is not as nice to test against. You either need to do:
```
expect(objA.equals(new SafeTypeThing('value i care about')).to.be.true;
```
or do as above and cast to a reduced value (e.g. 'value i care about', i.e. '147') to use the chai expectation functions.